instrumentation/apache-camel-2.20/apache-camel-2.20.gradle

```diff
@@ -0,0 +1,37 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.apache.camel"
+    module = "camel-core"
+    versions = "[2.20.1,3)"
+  }
+}
+
+dependencies {
+  library group: 'org.apache.camel', name: 'camel-core', version: '2.20.1'
+
+  testImplementation project(':instrumentation:apache-httpclient:apache-httpclient-2.0')
+  testImplementation project(':instrumentation:servlet:servlet-2.2')
+  testImplementation project(':instrumentation:servlet:servlet-3.0')
+
+  testImplementation group: 'org.spockframework', name: 'spock-spring', version: "$versions.spock"
+
+  testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: '1.5.17.RELEASE'
+  testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter', version: '1.5.17.RELEASE'
+
+  testImplementation group: 'org.apache.camel', name: 'camel-spring-boot-starter', version: '2.20.1'
+  testImplementation group: 'org.apache.camel', name: 'camel-spring-boot-starter', version: '2.20.1'
+  testImplementation group: 'org.apache.camel', name: 'camel-jetty-starter', version: '2.20.1'
+  testImplementation group: 'org.apache.camel', name: 'camel-http-starter', version: '2.20.1'
+  testImplementation group: 'org.apache.camel', name: 'camel-jaxb-starter', version: '2.20.1'
+  testImplementation group: 'org.apache.camel', name: 'camel-undertow', version: '2.20.1'
+
+  testImplementation 'javax.xml.bind:jaxb-api:2.3.1'
+
+  latestDepTestLibrary group: 'org.apache.camel', name: 'camel-core', version: '2.20.3'
```

Comment:
Why stop at 2.20.3? Won't thiswork for later 2.x versions?

Reply:
Haven't tested - can try latest 2.x (2.25.2)

Reply:
You can just remove this line if the latest one works